src/k8s_utils.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Canonical Limited
+# See LICENSE file for licensing details.
+
+"""Utility methods related to Kubernetes resources."""
+
+from lightkube import Client
+from lightkube.core.exceptions import ApiError
+from lightkube.resources.core_v1 import Namespace, ServiceAccount
+
+
+def is_valid_namespace(namespace: str) -> bool:
```

Comment:
**out-of-scope, refactoring** these functionalities seems quite good for a "k8s manager"

Reply:
Noted.